src/ComfortPay/Message/ListOfExpiredRequest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Omnipay\ComfortPay\Message;
+
+use SoapClient;
+use SimpleXMLElement;
+
+class ListOfExpiredRequest extends AbstractSoapRequest
+{
+    public function getExpDate()
+    {
+        return $this->getParameter('expDate');
+    }
+
+    public function setExpDate($value)
+    {
+        return $this->setParameter('expDate', $value);
+    }
+
+    public function getData()
+    {
+        $this->validate('expDate');
+
+        $data = parent::getData();
+        $data['expDate'] = $this->getExpDate();
```

Comment:
@tomaj Don't we want to validate `YYYYMMDD` format? Or is it better to let bank validate/fail on it?